client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -75,7 +75,23 @@
 }
 
 .editor-simple-payments-modal__list {
-	padding: 8px 16px;
 	flex: auto;
 	overflow-y: auto;
 }
+
+.editor-simple-payments-modal__list-item {
+	display: flex;
+	align-items: center;
+}
+
+input[type="radio"].editor-simple-payments-modal__list-radio {
```

Comment:
There must be some other way how to accomplish the same. :) /cc @mtias, our styling guru.